src/com/facebook/buck/cxx/toolchain/objectfile/Machos.java

```diff
@@ -65,7 +68,6 @@ static void setUuid(MappedByteBuffer map, byte[] uuid) throws MachoException {
       }
     }
 
-    throw new MachoException("LC_UUID command not found");
```

Comment:
The point of this method is to scrub non-deterministic metadata found in the LC_UUID command in the binary. If that command is not present (which it is not in cgo binaries), then nothing needs to be scrubbed, it shouldn't fail if it's not found.